### PR TITLE
Adding issue_template.md

### PR DESCRIPTION
**Pull request by Fiona-Waters**
Resolves https://github.com/project-codeflare/.github/pull/14
Adding a template for our github issues.

### CODE REVIEW

**Review by z103cb — COMMENTED** [review]
What is an "issue"? Is it a feature request or a bug report ?. If is a bug request what do you think of this template:

```md
### Describe the Bug

A clear and concise description of the bug.

### Steps to Reproduce the Bug

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

### What Have You Already Tried to Debug the Issue?

Provide a clear and concise description of how you tried to debug this issue.

### Expected Behavior

A clear and concise description of what you expected to happen instead.

### Screenshots, Console Output, Logs, etc.

Add screenshots of UIs (like dashboards), etc. that help explain the issue.

### Affected Releases

List any release versions, git commit hashes, or git tags, etc. that you know show the bug. If it is the latest `HEAD` on `main`, just put that.

### Additional Context

Add as applicable and when known:

* OS: 1) MacOS, 2) Linux, 3) Windows, 4) iOS, 5) Android: [1 - 5]
* OS Version: [e.g. RedHat Linux X.Y.Z, MacOS Monterey, ...]
* Browser (UI issues): 1) Chrome, 2) Safari, 3) Firefox, 4) Other (describe):  [1 - 4 + description?]
* Browser Version (UI issues): [e.g. Firefix 97.0]
* Cloud: 1) AWS, 2) IBM Cloud, 3) Other (describe), or 4) on-premise: [1 - 4 + description?]
* Kubernetes: 1) OpenShift, 2) Other K8s [1 - 2 + description]
* OpenShift or K8s version: [e.g. 1.23.1]
* Other relevant info

Add any other information you think might be useful here.
```

**Comment by Fiona-Waters** [comment]
@z103cb I think that's a great idea and very thorough template which I really like. We can add multiple templates. I think then maybe we could have a bug issue template like the one you have proposed, a feature request template and a general template for anything that doesn't fit the other 2. The one I have added would be the general one. Or maybe we only need 2. What do you think?

**Comment by z103cb** [comment]
> @z103cb I think that's a great idea and very thorough template which I really like. We can add multiple templates. I think then maybe we could have a bug issue template like the one you have proposed, a feature request template and a general template for anything that doesn't fit the other 2. The one I have added would be the general one. Or maybe we only need 2. What do you think?

I think it is ok to have three templates, but I will leave it to others for a final call.

Here's a feature request sample we have used in the past. I am _stealing_ from the work that @deanwampler has done.

```md
### Name of Feature or Improvement

TL;DR of what you are requesting...

### Description of Problem the Feature Should Solve

A clear and concise description of the problem the proposal would address. E.g., "I'm frequently frustrated when X happens." "It's too tedious/confusing to do Y."

### Describe the Solution You Would Like to See

Description of the proposed solution.

### Describe Alternatives You Have Considered

Description of any alternative solutions or features you have considered.

### Additional Context

Add any other context, screenshots, console logs, etc. about the request here.
```

**Comment by dimakis** [comment]
I'm not that familar with how the templates here work. So if we classify the issue as a bug (via label?) then the template changes? is this how we'd have multiple templates?

I think having the three is a good idea, I like all three.

**Review by anishasthana — APPROVED** [review]
